src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2710,6 +2710,31 @@ theorem toNat_mul_of_lt {w} {x y : BitVec w} (h : x.toNat * y.toNat < 2^w) :
     (x * y).toNat = x.toNat * y.toNat := by
   rw [BitVec.toNat_mul, Nat.mod_eq_of_lt h]
 
+
+/--
+`(a ≤ b + c)` if and only if `(a - c ≤ b)`
+when `a - c` and `b + c` do not overflow.
+-/
+theorem le_add_iff_sub_le {a b c : BitVec w₁}
+   (hac : c ≤ a) (hbc : b.toNat + c.toNat < 2^w₁) :
```

Comment:
```suggestion
theorem le_add_iff_sub_le {x y z : BitVec w}
   (hac : z ≤ x) (hbc : y.toNat + z.toNat < 2^w) :
```

Reply:
Update the comments and other theorem accordingly.

Reply:
Done.